src/portal_visualization/builders/anndata_builders.py

```diff
@@ -51,6 +52,12 @@ def get_conf_cells(self):
                 # If the dataset didn't have Azimuth annotations, it would be b'\x00'.
                 cell_set_obs.append("predicted.ASCT.celltype")
                 cell_set_obs_names.append("Predicted ASCT Cell Type")
+        # Check for an alias for gene names to add to the view config.
+        try:
+            gene_alias = 'var/hugo_symbol'
```

Comment:
```suggestion
        gene_alias = 'var/hugo_symbol'
        try:
```
It really doesn't matter in this case, but the idea is to only wrap code with `try` that really needs it... The more code that's wrapped in the try, the less you know about what precisely failed.